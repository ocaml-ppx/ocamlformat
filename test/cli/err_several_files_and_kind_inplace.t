  $ touch a.mli b.ml

  $ ocamlformat --impl --check a.mli b.ml
  ocamlformat: Cannot specify --impl or --intf with multiple inputs
  [1]
