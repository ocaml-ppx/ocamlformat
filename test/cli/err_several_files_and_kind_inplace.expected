ocamlformat: Cannot specify --impl or --intf with multiple inputs
